openlibrary/i18n/fr/messages.po

```diff
@@ -329,7 +332,7 @@ msgstr "Publication la plus ancienne"
 #: openlibrary/templates/work_search.html:114 view.html:141 view.html:143
 #: view.html:145 view.html:147
 msgid "Most Recent"
-msgstr "Le plus récent"
+msgstr "La plus récente"
```

Comment:
The plural form would be perfectly correct here.

For `Publication la plus ancienne`, I was afraid we loose the sense here. But the only dates shown in the results are the first publication dates, so we could safely go for `Les plus anciennes`.
I was actually confused because when you sort by `First Published` and you have results without first publication date, they show up first and you don't understand what the sort option did. I'll check if there is an issue opened on that matter, as I believe first published works should appear first and not works with unknown published dates.

How do you proceed then to add new commits? Do I continue on my branch and update this PR or do new commits requires a new PR?

Reply:
👍 sounds good! New branch, new commits, new pr. I was kind of assuming I would be wrong 😅